tools/rack/rack.plugin/plugin.xml

```diff
@@ -497,15 +737,23 @@
       type="java.lang.Object">
     </propertyTester>
   </extension>
-  <extension id="com.ge.research.rack.product" name="RITE"
-    point="org.eclipse.core.runtime.products">
-    <product name="RITE"
-      application="org.eclipse.ui.ide.workbench">
-    </product>
-    <property name="appName" value="RITE" />
-    <property name="windowImages"
-      value="./RITE-16.bmp,./RITE-32.bmp,./RITE-128.bmp,./RITE-256.bmp" />
```

Comment:
Not everything is an addition.  This extension was deleted.  Did you really want to remove the RITE image from the application splash box?

Reply:
The problem was a bad merge sometime ago that resulted in a lot of duplicate material in the plugin.xml.
The item you noted was repeated elsewhere in the file.
So I did detailed diffing and reordered some material to make it clear all the changes are additions -- and inspected them to be sure they are all additions in support of this PR. Should be good now. (and tested)